### PR TITLE
Adding windows forwarded events connector ID

### PR DESCRIPTION
**Pull request by yaelrbergman**
Required items, please complete
   
   Change(s):
Adding windows forwarded events connector ID

   Reason for Change(s):
WindowsForwardedEvents is needed 
   Version Updated:
No

   Testing Completed:
Yes

   Checked that the validations are passing and have addressed any issues that are present:
yes

### CODE REVIEW

File: Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml [hunk 1]
```diff
@@ -60,7 +60,11 @@ requiredDataConnectors:
       - Syslog
   - connectorId: GCPDNSDataConnector
     dataTypes: 
+      - SecurityEvents
```
Comment (oshezaf) [inline Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml:63]: I think you missed something here.

File: Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml [hunk 2]
```diff
@@ -190,6 +194,10 @@ entityMappings:
     fieldMappings:
       - identifier: FullName
         columnName: HostCustomEntity
+  - entityType: IP
```
Comment (oshezaf) [inline Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml:197]: How did the new entity got in?

File: Detections/MultipleDataSources/ZincJan272021IOCs.yaml [hunk 3]
```diff
@@ -51,11 +51,14 @@ requiredDataConnectors:
       - Cisco_Umbrella_dns_CL
   - connectorId: ASimDnsActivityLogs
     dataTypes: 
-      - ASimDnsActivityLogs
+          - ASimDnsActivityLogs
   - connectorId: Corelight
     dataTypes: 
       - Corelight_CL
-
+      - SecurityEvents
```
Comment (oshezaf) [inline Detections/MultipleDataSources/ZincJan272021IOCs.yaml:58]: Again, seems like Security Events are in the wrong place.

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml (hunk 2 above)
Comment (yaelrbergman) [reply Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml:197]: @oshezaf it's from a conflict with master

**Review by oshezaf — APPROVED** [review]

**Pull request merged**